Meshtastic/Views/Nodes/TraceRouteLog.swift

```diff
@@ -37,14 +37,19 @@ struct TraceRouteLog: View {
 				VStack {
 					List(node.traceRoutes?.reversed() as? [TraceRouteEntity] ?? [], id: \.self, selection: $selectedRoute) { route in
 						Label {
-							if route.response && route.hopsTowards == 0 {
+							if route.response && route.hopsTowards == 0 && route.hopsBack == 0 {
 								Text("\(route.time?.formatted() ?? "unknown".localized) - Direct")
 									.font(.caption)
-							} else if route.response && route.hopsTowards == 1 {
-								Text("\(route.time?.formatted() ?? "unknown".localized) - 1 Hop")
+							} else if route.response && route.hopsTowards == route.hopsBack {
+								let hopLabel = route.hopsTowards == 1 ? "Hop" : "Hops"
```

Comment:
Question.
Shouldn't this also have .localized?

`== 1 ? "Hop".localized : "Hops".localized`

Thanks up front!

Reply:
Thanks for the note. Yes, this also lead me down a path to utilize the swift localization for pluralizing.